src/applications/gi/sass/partials/_gi-profile-page.scss

```diff
@@ -1,4 +1,24 @@
 .gi-app .profile-page {
+  .vettec-logo-container {
+    text-align: right;
+  }
+  .vettec-logo {
+    height: 85px;
+    width: 179px;
+    padding-bottom: 8px;
```

Comment:
While I think it makes sense to have `width` and `height` in `px`, I typically prefer paddings to be in `em` myself. I don't remember exactly why I started, but I think it had something to do with zooming and scaling up the font size. Something something mobile devices too?

I definitely won't say this is wrong like this or anything, but I generally recommend `em`. Maybe try zooming and looking at it on a mobile device to see if it still looks like you expect it to.